mmdet3d/datasets/s3dis_dataset.py

```diff
@@ -24,133 +23,127 @@ class S3DISDataset(Det3DDataset):
     Args:
```

Comment:
The docstring in the line 21: `mmengine.datasets.dataset_wrappers.ConcatDataset`